pom.xml

```diff
@@ -53,6 +53,18 @@
       <groupId>org.springframework.boot</groupId>
       <artifactId>spring-boot-starter-test</artifactId>
       <scope>test</scope>
+      <exclusions>
```

Comment:
The exclusion of the vintage junit engine is not necessary as it is no longer included. Please confirm this with `mvn dependency:tree`